ac_c.lua

```diff
@@ -54,6 +54,27 @@ Citizen.CreateThread(function()
 	end
 end)
 
+Citizen.CreateThread(function()
+	while true do
+		Citizen.Wait(60000)
+		if NetworkIsInSpectatorMode() then
+			TriggerServerEvent("AntiCheese:Misc", "Cheating", "Spectator", true)
```

Comment:
I would recommend making seperate event for these as you are reporting what exactly it does anyway, having them grouped into one "generic" event just makes them harder to customize.